### PR TITLE
[IN-60] Dispatch hookup for matchesReducer & Feedback Component

### PR DESCRIPTION
**Pull request by ghost**
#### User Story
IN-60

#### Changes made
- Hooked up dispatch so that on continue button click in page 4, state updated
- Fixed matching functions
- Feedback component grabs and renders state information
- One issue: Feedback component not properly placed on Review page

#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
Describe the test steps to test your changes.

### CODE REVIEW

**Comment by tommytwm** [comment]
I'm not particularly sure how to test this. Is testing something `TODO` in the future when we have test data?

**Review by tommytwm — APPROVED** [review]

**Review by ReneReid — APPROVED** [review]

**Comment by tommytwm** [comment]
Resolve merge conflicts

**Comment by ghost** [comment]
Currently blocked due to testing; need to resolve discrepancy between back-end and this filter so we do get matches! Will need to fix & merge after student schema added to back-end (Rene’s story).

**Comment by ghost** [comment]
Just updated it on merge with dev, now works on a database of 1000 students, and makes use of getRequests to get all those students from the mongodb

**Review by tommytwm — APPROVED** [review]

File: internify/src/components/organisms/Feedback.js [hunk 1]
```diff
@@ -16,32 +24,38 @@ const Feedback = () => {
...
 
+  // calculate seeking students
+  function seekingStudents(totalStudents) {
+    let seekingLength = 0;
+    for (let i = 0; i < totalStudents.length; i++) {
+      if (totalStudents[i].seeking) {
+        seekingLength++;
+      }
+    }
+    console.log(seekingLength);
```
Comment (JudeShamsi) [inline internify/src/components/organisms/Feedback.js]: can you remove this

File: internify/src/store/reducers/matchesReducer.js [hunk 2]
```diff
@@ -22,95 +23,191 @@ export default function matchesReducer(state = initialState, action) {
...
+      matchCitizen(student.candidate, posting.candidates)
   );
 
-  const id = posting.id;
+  const jobId = posting.jobId;
   let match = {};
-  match[id] = matchedStudents;
+  match[jobId] = matchedStudents;
+
+  console.log(match);
```
Comment (JudeShamsi) [inline internify/src/store/reducers/matchesReducer.js]: remove

File: internify/src/store/reducers/matchesReducer.js [hunk 3]
```diff
@@ -9,8 +9,9 @@ export default function matchesReducer(state = initialState, action) {
     case PROCESS_MATCHES:
       const students = action.payload.students;
       const posting = action.payload.posting;
+      console.log(posting);
```
Comment (JudeShamsi) [inline internify/src/store/reducers/matchesReducer.js]: remove

**Review by JudeShamsi — COMMENTED** [review]
Awesome job dude

**Review by JudeShamsi — APPROVED** [review]

**Pull request merged**